submissions/Payouts_integration/integration_docx.html

```diff
@@ -0,0 +1,119 @@
+<p>Digital Services: Pay content creators, influencers, or contributors effortlessly.</p>
+<p>Non-profit Organisations: Distribute funds to beneficiaries or support initiatives.</p>
+
+<p><b>Finished product: </p></b>
+<a href="https://rb.gy/y5awd"> currency converter using chimoney</a>
+
+<p> Integration Steps (Using javascript and html as an Example): </p>
+
+
+<p><b>Step 1: Obtain API Key</b></p>
+<p>Register on Chimoney and obtain your API key in the developers portal </p>
+<p>Follow this tutorial on how to get the get started with chimoney api and developer portal</p>
+<p>For this tutorial we will be using the convert local currency amount to USD endpoint this is the url for that https://api.chimoney.io/v0.2/payouts/mobile-money
+</p>
+
+<p><b>Step 2: user interface</p></b>
+
+We will create a small interface for easier integration with our api key the interface will provide a way for us to pass the two parameters that is the amount and the country code that are required for the endpoint.
+
+<img src="./interface.png" alt="currency converter interface ">
+
+<xmp>
+<body>
+    <div class="title">
+        <h2>Get usd amount in Local</h2>
+    </div>
+    <div class="country">
+        sample country codes
+        <ul>
+            <li>Kenya code : KES</li>
+            <li>Nigeria code: NG</li>
+            <li>Ghana code: GHN</li>
+        </ul>
+    </div>
+    <div class="form">
+        <label>Country symbol</label> 
+        <input type="text" id="symbol"> <br/> <br/>  //input tags
+        <label>Amount  </label>  
+        <input type="text" id="amount"><br/><br/> //input tags
+    </div>
+        <div class="button">
+            <button onclick="convert()">Convert</button> //calls the convert function
+        </div>
+        <div id="result"></div>
+</body>
+</xmp>
+
+The major parts of this code is this line <xmp><button onclick="convert()"></xmp>Convert</button>
+Which on click calls the convert method which is where we have our api function
+
+
+
+<p><b>Step 3: Initialize Chimoney</p></b>
+
+<p></p>Let's now dive into the real part we will be using javascript for this and use the fetch in built method
+<p></p>function convert () { //function triggered on click the 
+    <p></p>const countrySymbol = document.getElementById('symbol').value; // this are to passed as parameter to the endpoint and the are keyed in by the user
+    <p></p>const amountInUSD = document.getElementById('amount').value;
+
+    <p>const options = { // fetch method takes to params that is the options and url so lets define the options
+        <p>method: 'GET', // this is the method used get method retrieves data 
+            <p>headers: { // takes in our api key as application json
+                <p>accept: 'application-json',
+                    <p>'X-API-KEY': "f53b14db7658d93aeb2f8589838b87e01d730d82b9c73e9878065452902790b7" //for best practices save your api key in the .env file
```

Comment:
You should not have this key in the frontend

Reply:
i passed it has an env variable. The one above is  for demo purposes for those who may not know how to use env